payment_utf8 /README.rst

```diff
@@ -0,0 +1,7 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Elico Account Extra
```

Comment:
@duanyp1991 Actually I am confused by this module: it states that it is to fix UTF8 but it seems to link paypal and accounting payments.
Module name and description should be fixed accordingly: can you help with that?